tests/kubernetes/bin/test_paasta_secrets_sync.py

```diff
@@ -369,3 +371,138 @@ def test_sync_boto_secrets():
         assert mock_update_secret.called
         assert mock_create_kubernetes_secret_signature.called
         assert not mock_update_kubernetes_secret_signature.called
+
+
+def test_sync_crypto_secrets():
```

Comment:
it'd be nice to split this and the other sync tests into multiple rather than testing every case in a single function body - but this is probably fine for now :p